src/lib/components/PostPreview/PostPreviewCard.tsx

```diff
@@ -1,129 +1,135 @@
 import React from 'react';
-import { useHistory } from 'react-router-dom';
+import { Link } from 'react-router-dom';
+import { ThemeProvider } from '@material-ui/core/styles';
 import Card from '@material-ui/core/Card';
 import Box from '@material-ui/core/Box';
 import CardContent from '@material-ui/core/CardContent';
 import CardMedia from '@material-ui/core/CardMedia';
 import Typography from '@material-ui/core/Typography';
 import { IPost } from '../../types';
 import { useStyles } from '../../styles/PostPreviewCard.styles';
-import PostDirectionLink from '../PostDirectionLink';
+import { MAIN_THEME } from '../../theme/theme';
 
 export interface IPostPreviewCardProps {
   data: IPost;
 }
-
 const PostPreviewCard: React.FC<IPostPreviewCardProps> = (props) => {
   const { data } = props;
-  const history = useHistory();
-
+  let expertLink = ``;
+  const postLink = `/posts/${data.id}`;
   let authorFullName = '';
+
   if (data.author?.firstName && data.author?.lastName) {
     authorFullName = `${data.author?.firstName} ${data.author?.lastName}`;
+    expertLink = `/experts/${data.author.id}`;
   }
 
-  const goPostView = () => {
-    history.push(`/posts/${data.id}`);
-  };
-
-  const goExpertPage = () => {
-    if (data.author) {
-      history.push(`/experts/${data.author.id}`);
-    }
-  };
-
   const classes = useStyles();
   return (
-    <Card className={classes.root}>
-      <Box
-        display="flex"
-        flexDirection="column"
-        justifyContent="space-between"
-        height="100%"
-      >
+    <ThemeProvider theme={MAIN_THEME}>
```

Comment:
only one theme provider app.tsx should be enough, pls double check

Reply:
repeated ThemeProvider components removed